.changeset/sour-hounds-shout.md

```diff
@@ -0,0 +1,41 @@
+---
+"@jpmorganchase/uitk-core": minor
+"@jpmorganchase/uitk-icons": minor
```

Comment:
shouldn't this just be `uitk-icons` ?

Reply:
I'm not entirely sure. Components which depended on Icon have also had to be updated in core and lab (so their api/html have changed) and we've added a new css variable to the theme

Reply:
true there

Reply:
So my thoughts on this are:

Core - Typescript types changes and components that depend on Icon changed
Icons - The changed directly happened here
Lab - Components that depend on Icon changed
Theme - new characteristics
Theme-editor - Can be removed from here